forge-gui-mobile/src/forge/adventure/world/BiomeStructure.java

```diff
@@ -0,0 +1,141 @@
+package forge.adventure.world;
+
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.github.sjcasey21.wavefunctioncollapse.OverlappingModel;
+import forge.adventure.data.BiomeStructureData;
+import forge.adventure.util.Config;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
```

Comment:
Does this class is only accessed via the desktop adventure editor? mobile version (android) don't have BufferedImage and ImageIO is not part of mobile platform on android.